test/unit/spec/MfaVerify_spec.js

```diff
@@ -523,36 +514,54 @@ function (Okta,
 
       });
       Expect.describe('Remember device', function () {
+
+        function setupWithMfaPolicy(options) {
```

Comment:
can we move this to somewhere in between lines 181-221? (probably after line 221) This is so all the setup functions are at the top of the test, so if we need to reuse them then they are easier to find

Reply:
Done